features/edit_vendor.feature

```diff
@@ -24,7 +24,7 @@ Feature: Edit an existing vendor in the database
     When I fill in "Name" with ""
     And I submit the form
     Then I should see an error message on the edit vendor page
-    And I should see the vendor attributes filled in
+    And I should see the vendor attributes, except "Name", filled in
```

Comment:
Why are we no longer filling in name?

Reply:
I changed `edit` to make the form keep the edited information if a validation fails. Since the user changed the name to be blank, the form should still show the name as blank when the validation fails.